cmd/exec.go

```diff
@@ -67,7 +64,7 @@ var execCmd = &cobra.Command{
 		}
 
 		// Execute the command using the resolved shell instance
-		output, err := shellInstance.Exec(args[0], args[1:]...)
```

Comment:
I believe that we still want to use Exec so that the output comes back to stdout